docs/source/upgrade/example/coinV2/daml.yaml

```diff
@@ -10,7 +10,7 @@ dependencies:
   - daml-prim
   - daml-stdlib
 data-dependencies:
-  - ../coinV1/.daml/dist/coin-1.0.0.dar
+  - coin-1.0.0.dar
```

Comment:
It didn't make much sense to say `../coinV1/...` either, as the documentation never mention the layout of the projects in the directory structure.

Reply:
I think it should do that then :slightly_smiling_face: Users should be able to reproduce this. The most obvious way is to put the projects right next to each other and in that case this works.

Reply:
I think the documentation is improved by loosing the dots!
Perhaps we could take the popular approach or writing `path/to/coin-1.0.0.dar`
And fix the rule instance to match.

Reply:
I’m fine with `path/to/coin-1.0.0.dar` provided we extend the documentation that you need to edit this and suggest `../coinV1/.daml/dist/coin-1.0.0.dar` which at least for development is the most convenient option since it removes the need to copy around files.

Reply:
Ok, I'll do this. Thanks!

Reply:
this is done

Reply:
Thanks a lot! Looks great :+1: